src/main/scala/com/fulcrumgenomics/vcf/api/Variant.scala

```diff
@@ -24,6 +24,8 @@
 
 package com.fulcrumgenomics.vcf.api
 
+import com.fulcrumgenomics.vcf.api.VcfConversions.VariantLocatable
```

Comment:
This doesn't belong here.  Note that `VcfConvefsions` is `private[api]` and is documented as follows:

```
  * Object that provides methods for converting from fgbio's scala VCF classes to HTSJDK's
  * Java VCF-related classes and vice-versa.  Only intended for internal use within the `api`
  * package and should not be exposed publicly.
```